Configuration/AlCa/python/autoCond.py

```diff
@@ -32,13 +32,13 @@
     # GlobalTag for Run2 HI data
     'run2_data_promptlike_hi'      : '123X_dataRun2_PromptLike_HI_v1',
     # GlobalTag for Run3 HLT: it points to the online GT
-    'run3_hlt'                     : '123X_dataRun3_HLT_v1',
+    'run3_hlt'                     : '123X_dataRun3_HLT_v2',
```

Comment:
is the update of this file intended in this PR?

Reply:
it's not, but there is no https://cmssdt.cern.ch/SDT/html/cmssdt-ib/#/ib/CMSSW_12_3_X 11:00 IB to which I could rebase to, yet....

Reply:
Never mind, I fetched the master release, it's not looking too much better tho, like this I have 3 commits

Reply:
ok thanks! that worked